tiledb/CMakeLists.txt

```diff
@@ -432,7 +442,8 @@ endif()
 
 # Win32 specific libraries
 if (WIN32)
-  set(WIN32_LIBS shlwapi rpcrt4 bcrypt)
+  #ws2_32.lib, crypt32.lib needed to satisfy curl dependencies
+  set(WIN32_LIBS shlwapi rpcrt4 bcrypt ws2_32 crypt32)
```

Comment:
These extra dependencies need to be added only if building with azure.

Reply:
Done.